Documentation/operations/upgrade-warning.rst

```diff
@@ -7,8 +7,12 @@
 .. warning::
 
    Read the full upgrade guide to understand all the necessary steps before
-   performing them.
+   performing them. Pay particular attention to :ref:`current_release_required_changes`
+   before proceeding.
 
-   Do not upgrade to \ |NEXT_RELEASE|.0 before reading the section
-   :ref:`current_release_required_changes` and completing the required steps.
-   Skipping this step may lead to an non-functional upgrade.
+   The only tested rollback path is between consecutive minor releases. This means
+   that going from (a hypothethical) 1.42 to 1.43 and back is supported while going
+   to 1.44 is not.
```

Comment:
Couple of suggestions here. Let me know what you think:

* This statement is also relevant to upgrade paths
* Use smaller version numbers, as these are commonly associated with examples
* Provide a clear instruction to users given this information

```suggestion
   The only tested rollback and upgrade path is between consecutive minor releases.
   Always perform rollbacks and upgrades between one minor release at a time.
   This means that going from (a hypothetical) 1.1 to 1.2 and back is supported
   while going from 1.1 to 1.3 and back is not.
```